src/quick-lane/views/index.ts

```diff
@@ -0,0 +1 @@
+export { default as Detail } from './QuickLaneDetail';
```

Comment:
Exporteer deze ook maar als `QuickLaneDetail` dan blijven we in lijn met de rest van de app.